metisp/pymetis/src/pymetis/recipes/ifu/metis_ifu_rsrf.py

```diff
@@ -95,13 +108,13 @@ def add_properties(self):
             self.properties.append(self.header)
 
 
-    class ProductRsrfIfu(PipelineProduct):
+    class ProductRsrfIfu(TableProduct):
         _tag: str = r"RSRF_IFU"
         group = cpl.ui.Frame.FrameGroup.CALIB # TBC
```

Comment:
I think it should be `PRODUCT` for all products. One thing I notice is that checksums (`DATAMD5`) are not computed unless `FrameGroup.PRODUCT` is used.